model/atmosphere/dycore/tests/dycore_tests/test_solve_nonhydro.py

```diff
@@ -119,6 +119,8 @@ def test_nonhydro_predictor_step(
     ndyn_substeps,
     caplog,
 ):
+    if (backend is not None) and ("gtfn_cpu" in backend.name):
+        pytest.skip("Temporary skip due to gt4py offset error output")
```

Comment:
Also I am curious: which stencil does cause the problem? And why does it show up now? Is this a change of ours or of gt4py?

Reply:
`compute_tangential_wind`